ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos.hs

```diff
@@ -729,12 +729,12 @@ instance
           Views.lvProtocolVersion = SL.ccProtocolVersion lvChainChecks
         }
       where
-        coercePoolDistr :: SL.PoolDistr c1 -> SL.PoolDistr c2
+        coercePoolDistr :: SL.PoolDistr -> SL.PoolDistr
         coercePoolDistr (SL.PoolDistr m totalActiveStake) =
```

Comment:
`coercePoolDistr` can be removed, since all they did was change the type of crypto parameter, which is now gone